lib/yabeda/porta.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+# Based on https://github.com/yabeda-rb/yabeda-rails
+
+require "yabeda"
+
+module Yabeda
+  module Porta
+    LONG_RUNNING_REQUEST_BUCKETS = [
+      0.005, 0.01, 0.05, 0.1, 0.5, 1, 5, 10, # standard
+      30, 60, 600, # Sometimes requests may be really long-running
```

Comment:
Don't we have some 30-40 seconds request timeout? Fine with me, just asking.

Reply:
Yeah, true, `UNICORN_TIMEOUT` defaults to `40`
maybe on this line we can keep `40` and `600` (for anything that for some reason is not within `40` - maybe the timeout was manually overriden, or whatever)